README.md

```diff
@@ -39,12 +39,10 @@ If you want to work on BrainBox's code, you'll need a local installation:
 
 1. install and start `mongo` database
 2. clone the repo and `cd` to the brainbox directory
-3. `mkdir public/data`
 4. [create a new OAuth application](https://github.com/settings/applications/new) for your local brainbox url (http://localhost:3000 by default)
```

Comment:
Note: I don't change the numbered list because markdown will [ignore the numbers and automatically order them](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#lists). But let me know if you preferer it ordered in the file :)